QA/manual_testing.md

```
@@ -60,7 +61,32 @@
 2. 確認事項:
    - 環境変数が正常にエクスポートされること
 
-#### 2.4 新しいbashセッションの起動
+#### 2.4 TUIを使用した選択的エクスポート
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)